packages/office-ui-fabric-react/src/components/Slider/Slider.styles.ts

```diff
@@ -22,21 +22,22 @@ const GlobalClassNames = {
 
 export const getStyles = (props: ISliderStyleProps): ISliderStyles => {
   const { className, titleLabelClassName, theme, vertical, disabled, showTransitions, showValue } = props;
-  const { palette } = theme;
+  const { palette, semanticColors } = theme;
   const classNames = getGlobalClassNames(GlobalClassNames, theme);
 
   // Tokens
-  const sliderInteractedActiveSectionColor = palette.themePrimary;
-  const sliderInteractedInactiveSectionColor = palette.themeLight;
-  const sliderRestActiveSectionColor = palette.neutralSecondary;
-  const sliderRestInactiveSectionColor = palette.neutralTertiaryAlt;
+  const sliderInteractedActiveSectionColor = semanticColors.inputBackgroundCheckedHovered;
```

Comment:
what does "interacted" mean?

Reply:
Interacted means its being interacted with, so either hovered on or pressed and `sliderInteractedActiveSectionColor` specifically  is the interacted state pressed. 
<img width="545" alt="Screen Shot 2019-08-12 at 6 38 36 PM" src="https://user-images.githubusercontent.com/38991459/62909199-79bcb180-bd30-11e9-825a-3b2f2d594bf8.png">

Reply:
In terms of naming, we always say "pressed" instead of "active", even though "active" is what CSS uses. "pressed" makes more sense so we standardized on that.

Reply:
ah nm, "active" means the selected section here, that makes sense. you might want to explain the vocab in a comment
also, you don't need to prepend "slider" to every one of these since we already know we're in the slider component

Reply:
What about if I change it from `interacted` to `selected`?